actix-identity/src/identity_error.rs

```diff
@@ -0,0 +1,62 @@
+use actix_session::{SessionGetError, SessionInsertError};
+use actix_web::cookie::time::error::ComponentRange;
+
+/// Possible errors which can emerge from storing and retrieving identities.
+#[derive(Debug)]
```

Comment:
You could add the `#[non_exhaustive]` tag, so we can add new errors latter without breaking libraries using it

Reply:
Sweet - I've split the errors, but put `#[non_exhaustive]` wherever I left an enum